app/angular/src/client/preview/angular-beta/StorybookModule.test.ts

```diff
@@ -104,13 +112,15 @@ describe('StorybookModule', () => {
       it('should change inputs if storyProps$ Subject emit', async () => {
         const initialProps = {
           input: 'input',
+          inputBindingPropertyName: '',
```

Comment:
If you are wondering why I am modifying the test here :

If a new entry was added to the list of props between the initialisation of the story and a lazy (emit on storyProps$) change of the input values 
like : Initial `{ InputA : 'A' }` new change : `{ InputA : 'A', InputB: 'B' }`
The new Input was not added in the template like the first one `<foo [InputA]="InputA" ></foo>` because not new rendering a made only rxjs event with new props.
Before it worked because the property was added as a non Input Output property, overwriting the component property directly.
which was not good because angular could not detect the changes with the NgOnChanges for example

But anyway all this should not happen because a [higher level mechanism](https://github.com/storybookjs/storybook/blob/cf4ec52fbdded1527c5e94341f29f9b162fc2af1/app/angular/src/client/preview/angular-beta/AbstractRenderer.ts#L174) detect if lazy rendering can be done. By comparing templates to see if there is a diff. 

so in the test here all Input Output must be existing at init step